lib/hooks.js

```diff
@@ -0,0 +1,43 @@
+'use strict'
+
+const hooks = {
+  onRequest: [],
+  onRequestRaw: []
+}
```

Comment:
is this a global? It seems all this object should be built for each fastify instance.